awx/main/scheduler/dependency_graph.py

```diff
@@ -88,8 +88,7 @@ def mark_workflow_job(self, job):
         self.data[self.WORKFLOW_JOB_TEMPLATES_JOBS][job.workflow_job_template_id] = False
 
     def can_project_update_run(self, job):
-        return self.data[self.JOB_PROJECT_IDS].get(job.project_id, True) and \
```

Comment:
just to clarify, I think we should remove _all_ references to the variable `JOB_PROJECT_IDS` because it's no longer needed.

Reply:
removed!